pkg/config/config.go

```diff
@@ -0,0 +1,58 @@
+package config
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/red-hat-storage/managed-fusion-fleet-reconciler/pkg/db"
+	"gopkg.in/yaml.v2"
+)
+
+type Worker struct {
+	Count int `yaml:"count"`
+}
+
+type General struct {
+	Development bool `yaml:"development"`
+}
+type Config struct {
+	DB      db.DB   `yaml:"db"`
+	Worker  Worker  `yaml:"worker"`
+	General General `yaml:"general"`
+}
+
+func GetConfig(filePath string) (*Config, error) {
```

Comment:
Use a more descriptive and complete name:
```suggestion
func LoadAndValidateConfig(filePath string) (*Config, error) {
```